### PR TITLE
[support] Typescript for Polkadot identicon

### PR DESCRIPTION
**Pull request by henri-ly**
### 📝 Description

Fix typescripting for LLM and LLD of polkadot icon

### ❓ Context

- **Impacted projects**: `LLD LLM` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-7622 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by hedi-edelbloute — CHANGES_REQUESTED** [review]
Please explain why we don't cross the text coverage checkbox (atleast it's not a copy pasta @haammar-ledger )

File: apps/ledger-live-desktop/src/renderer/families/polkadot/Nomination/Row.tsx [hunk 1]
```diff
@@ -19,6 +18,7 @@ import ClockIcon from "~/renderer/icons/Clock";
 import ExclamationCircle from "~/renderer/icons/ExclamationCircle";
 import ToolTip from "~/renderer/components/Tooltip";
 import ExternalLink from "~/renderer/icons/ExternalLink";
+import Identicon from "@polkadot/react-identicon";
 const Wrapper = styled.div`
```
Comment (lvndry) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/Nomination/Row.tsx:22]:
```suggestion
import Identicon from "@polkadot/react-identicon";

const Wrapper = styled.div`
```

File: apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/fields/ValidatorRow.tsx [hunk 2]
```diff
@@ -11,6 +10,7 @@ import Text from "~/renderer/components/Text";
 import CheckBox from "~/renderer/components/CheckBox";
 import Tooltip from "~/renderer/components/Tooltip";
 import ExternalLink from "~/renderer/icons/ExternalLink";
+import Identicon from "@polkadot/react-identicon";
 const IconContainer = styled.div`
```
Comment (lvndry) [inline apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/fields/ValidatorRow.tsx:14]:
```suggestion
import Identicon from "@polkadot/react-identicon";

const IconContainer = styled.div`
```

**Review by lvndry — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Review by mcoursier-ledger — APPROVED** [review]

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**